src/commands/gate/__tests__/evaluate.test.ts

```diff
@@ -40,6 +43,9 @@ describe('evaluate', () => {
       const response: EvaluationResponse = {
         status: 'passed',
         rule_evaluations: [],
+        metadata: {
+          wait_time_ms: 0,
+        },
```

Comment:
Is this needed? If so, why? We should be able to handle responses without it no (which should happen in the normal case)?

Reply:
It's because of ts interface, it fails when running tests without it, but when running the command it works :D 
I'll try to fix it.